lib/prosperworks/company.rb

```diff
@@ -1,5 +1,6 @@
 module ProsperWorks
   class Company < BaseEntity
+    extend ApiOperations::Search
```

Comment:
Don't need to include this here, since `search` is already included on the `BaseEntity`